### DISCUSSION TITLE
トラックボールの動作不良について

### DISCUSSION THREAD

**Discussion by Sheepspider** [Yowkees/keyball#282] [Q&A]
ビルドガイドに従って右手側にトラックボールを接続したのですが、カーソルがボールの動きと関係ない方向に勝手に動作する、カーソルが突然ワープする、と言った現象が発生してしまいます。

恐らく#234、#238の方と同じ状況ですが、自力での解決が見込めません。

はんだ付けした基盤の写真は以下の通りです。
御手数ですが、わかることがあれば教えていただきたく存じます。

![16899947470944900790520668272616](https://github.com/Yowkees/keyball/assets/109937920/cb69cae1-1513-4eea-ac73-5d66ef51f902)
![16899947820804727770379851403580](https://github.com/Yowkees/keyball/assets/109937920/6d0e1ead-0c1a-480f-86ac-9c251d821f31)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
ICの方向等も問題ありませんし、とても綺麗に半田付けされているので、これで問題が起こるとは考えにくいですね。

ProMicroの初期不良はないでしょうか？
具体的には左右のProMicroを入れ替えて、念のためファームウェアを書き直しても改善しないかご確認お願いします。実施済でしたらすみません。

以上、お手数ですがご確認お願いいたします。

**Comment by Sheepspider** [comment]
迅速な回答ありがとうございます。

左右のPromicroを入れ替えてファームウェアを再度書き込みなおしましたが、状況は全く変わりませんでした。

参考になるかわかりませんが、トラックボールをスムーズに動かしている際のカーソルの挙動を録画いたしました。
特に左に移動する際に、斜め下にカーソルが飛ぶ挙動が多く見られます。
https://github.com/Yowkees/keyball/assets/109937920/3d694d3c-4532-42b9-bbfb-8898295ee912

**Comment by Yowkees** [comment]
動画のご送付ありがとうございます。
うーむなるほど。ちょっと私は経験したことのないカーソルの動きをしているようです。

実機を確認する必要があるかも知れませんので、一度、下記の白銀ラボウェブショップよりご連絡いただけますでしょうか。
https://shirogane-lab.net/inquiry

今後の対応をお話しさせていただければと思います。
お手数ですがよろしくお願いいたします。

**Comment by Sheepspider** [comment]
先日お問い合わせより連絡し、御指示の下、手持ちのKeybakk61を白金ラボ様に送付いたしました。
その後いかがでしょうか。
原因など把握できた際、ご連絡いただければ嬉しいです。

**Comment by Yowkees** [comment]
今週月曜日と水曜日にメールをお送りしております。迷惑メール等に入っていないかご確認をお願いいたします。
念のため、たった今Eメールを再送付いたしました。

**Comment by Sheepspider** [comment]
先ほどメールの方を確認いたしました。ありがとうございます。

重ねての質問になってしまうのですが、今回の件に関して各種費用に関する説明などは、いずこかに記載がありましたでしょうか。

利用規約や特定商取引法に関する表記等を全体的に拝見いたしましたが、調査・修理に関する記載が見つけられず、またメールでのやり取りについても、筐体の送付を希望するメールまでに請求や費用に関する文面が送料に関する文面のみであったため、「当方の負担はこちらからの送付にかかる発送料のみ」と読み取っていました。

今回の請求及び請求内容に関して、各種記載において当方の見落としがあった可能性がありますので、それらに関する記載が存在する文面がありましたら教えていただきたく存じます。

**Reply by Yowkees** [reply]
お世話になります。

「白銀ラボ」は自作キーボードキットとキーボードパーツの通販サイトです。
そのため、通信販売に関する規約のみ掲載しています。

本件のような修理サービスはご提供しておりません。

あくまで「動かない個所があるのでどうしても確認してほしい」という方向けにお客様ごとに個別で価格等を協議して対応するものです。
また、基板の製造不良等、当方の原因も考えられる事案については、ひとまず実機で原因を確認させていただき、初期不良であれば無償で対応させていただきます。組み立てミスであった場合には有償修理対応に切り替えさせていただき、費用を別途お見積りさせていただく対応をとる場合もございます。

キット販売という性質上「売って終わり」という対応をとるのは心苦しく、事案や不具合原因ごとに臨機応変に対応させていただいており、ご理解いただければ幸いです。

**Comment by Sheepspider** [comment]
ご返答ありがとうございます。
まず、今回の請求について了承するとともに、頂いたメールにおける「①現状での返却（＋調査費用のみの支払い）」で同意するものとし、筐体の送付と支払方法の提示をお願いいたします。


但し、今回の件につい疑問に思った点がございますので、一読下さると嬉しいです。

修理に関して只今いただいたメッセージについてはこちらとしても全く理解できるもので、請求に関しても修理費用及び部品費用についての提示は当方による決定権があり、疑義なく納得しております。
しかし調査費用の請求については、送付・調査の段階で費用が発生する旨の説明がないことから、此方としては完全に予想外且つ全く協議の余地のなかった請求と認識しております。

今後同様の調査・修理サービスを続けられる場合、金銭の請求及び請求内容に関わる不告知は、白銀ラボ様にとってトラブルを引き起こす火種になり得ることであると思われます。

素敵で便利な商品の提供と利用者への細やかな対応に大変感謝するとともに、
サービスの提供と対価に関する各種告知を適切に行うことを望みます。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

疑問点について、ご指摘ありがとうございます。
当方としては今回お送りしたメールはあくまでお見積りの段階でして、ご請求のつもりはございませんでした。
あくまで解析の段階でして、お見積りにご納得いただけないようでしたら、現状のままでのご返送も可能です。
協議の余地がないように感じたようでしたら、当方の文面に良くない部分があったと思い、深く反省しております。

みなさん修理完了を楽しみにしていただくことが過去に多かったので、少々、途中を端折ったメールとなってしまったように思います。
今後はもう少し段階を踏んで対応するように改善させていただきたく思います。

お預かりしているSheepspider様の実機について、今後の進め方についてはEメールにてご連絡させていただきたく、よろしくお願いいたします。ご連絡お待ちしております。

**Comment by Sheepspider** [comment]
ご連絡ありがとうございます

先刻お伝えした通りになりますが、①現状での返却(調査費用のみの支払い)を希望いたします。



Githubでも申し上げた点ですが、修理ないし調査の完了を待ちわびていたことに違いはありません。

殊、調査費用に関して事前に告知がなく、且つ調査が既に完了した後になって見積がなされた為、当方に選択の余地が無くなっていたことについての疑問でした。

即ち、実機を預かる件についてのメールの時点で、最低限調査費用がかかる旨を記載頂けるようお願いしたいということでございます。

調査いただいた内容についても、調査・修理に関するお見積もり金額についても、疑問や不安はございません。


どうかご理解いただけると幸いでございます。



2023年8月4日(金) 16:39 Yowkees ***@***.***>:

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

今後不具合調査の費用やお見積り、ご請求方法については事前連絡を徹底させていただきます。
ご迷惑をおかけしたにも関わらず、調査費用についてご理解をいただき本当にありがとうございます。

さきほどお支払い方法等についてメールでご連絡させていただきましたので、お手数ですがご確認お願いいたします。
以上、よろしくお願いいたします。
